recipe/meta.yaml

```diff
@@ -101,6 +110,11 @@ outputs:
         # host C compilers work fine (better) on mac
         - {{ compiler('c') }}
     test:
+      # Ensure the correct dependencies are available for testing
+      requires:      
+        - mpi 1.0 openmpi
+        - ucc  # [linux and not ppc64le]
+        - ucx  # [linux and not ppc64le]
```

Comment:
I may have missed something, but if you already added them to run (which is wrong, as noted above) why did you have to add them here again? This seems incorrect too.

Reply:
OK, let's remove these lines